x-pack/plugins/triggers_actions_ui/public/application/lib/action_variables.ts

```diff
@@ -5,14 +5,16 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import { AlertType, ActionVariable } from '../../types';
+import { ActionVariable, ActionVariables } from '../../types';
 
 // return a "flattened" list of action variables for an alertType
-export function actionVariablesFromAlertType(alertType: AlertType): ActionVariable[] {
+export function actionVariablesFromAlertType(actionVariables: ActionVariables): ActionVariable[] {
```

Comment:
nit: should this function name be updated since it's no longer taking `AlertType` as a parameter?